opal/runtime/opal_params.c

```diff
@@ -61,6 +61,10 @@ char *opal_timing_output = NULL;
 bool opal_timing_overhead = true;
 #endif
 
+#if OPAL_ENABLE_DEBUG
```

Comment:
Why is this defined here and not in one of the threads related file (possibly in mca/thread/base/) ?

Reply:
We moved this declaration to `opal/mca/threads/base/threads_base.c`.